draft-ietf-quic-tls.md

```diff
@@ -269,7 +271,7 @@ At a high level, there are two main interactions between the TLS and QUIC
 components:
 
 * The TLS component sends and receives messages via the QUIC component, with
-  QUIC providing a reliable stream abstraction to TLS.
+  QUIC providing a reliable stream and record abstraction to TLS.
 
```

Comment:
I'm not sure QUIC provides the record abstraction, at least not entirely, it only handles the record protection. TLS still does its own internal record framing as I understand it, but don't take my word for it.

Reply:
This is the big draft-14 change, is it not?

Reply:
Well, not sure about 14, possibly, but the big change was to remove TLS encryption layer piped through already encrypted QUIC. But as I understand it, TLS still maintains its own record structure and framing within the QUIC crypto stream. So in a sense it both is and isn't TLS records.

Reply:
The separation of CRYPTO into different spaces doesn't really result in a record-like abstraction.

Reply:
It's not the separate spaces, it's the crypto frame itself. We're stripping out the TLS record layer and replacing it with the frame. I'm not sure how you can compare the figures at the start of Sections 2 and 3 and come to a different conclusion.

Reply:
I don't want to hold up this PR forever on this point, but do @mt and @mikkelfj still disagree that QUIC is providing a record abstraction?

Reply:
I think that this would be a mistake.  TLS only requires a stream abstraction.  QUIC has its own need for a record-like abstraction (we call them packets), but that is not relevant here.

If you revert this, I can merge it.

(@martinduke, be careful who you are tagging...)